roles/filetree_create/tasks/team_access_list.yml

```diff
@@ -0,0 +1,62 @@
+---
+- name: "Get object id from API"
+  when: object_name is defined
+  ansible.builtin.set_fact:
+    object_id: "{{  (query(controller_api_plugin, 'api/v2/{{ object_type }}',
```

Comment:
This is not correcly formed, so it should be something like `'api/v2/' + object_type`. The same should apply to the other files.

Reply:
Done.